plugins/android_boot/README.md

```diff
@@ -0,0 +1,51 @@
+# Android Bootloaders
+
+## Introduction
+
+This plugin is used to update hardware that use partitions to store their firmware on.
+
+## Firmware Format
+
+The daemon will decompress the cabinet archive and extract a firmware blob as a Raw Disk File
+in the IMG format. The firmware blob will be flashed to the partition.
+
+This plugin supports the following protocol ID:
+
+* be.dylanvanassche.android_boot
+
+## GUID Generation
+
+The GUID is generated by combining the partition UUID of the block device, its label and optionally boot slot
+when using an Android A/B partitioning scheme, e.g.
+
+* `DRIVE\UUID_c49183ed-dddc-sd55-960b-64450fdsfnsd&LABEL_label&SLOT_a`
```

Comment:
64450fdsfnsd isn't a GUID...

Reply:
I took mine and random changed characters as I wasn't sure it was safe to publish this information? I can update it with the real one I have for the SHIFT6mq (see quirks)